features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/Environments.jsx

```diff
@@ -230,10 +230,18 @@ function Environments(props) {
                         </Paper>
                     </>
                 )}
+                {!selectedEndpoint && (api.lifeCycleStatus !== 'PROTOTYPED') && (
+                    <Typography variant='subtitle2' gutterBottom align='left' className={classes.sectionTitle}>
+                        <FormattedMessage
+                            id='Apis.Details.Environments.label.noendpoint'
+                            defaultMessage='No endpoints yet.'
+                        />
+                    </Typography>
+                )}
                 <GoToTryOut />
             </Box>
             <Box ml={8} alignItems='center' mt={1}>
-                {getDefaultVersionUrl() && (
```

Comment:
Need to check without making an API default version.

Reply:
Shall we revisit this later?

Reply:
+1, Since this is a blocker

Reply:
defaultVersionURLs is a child of endpointURLs ( selectedEndpoint). So Checking selectedEndpoint is the correct way to do this.